### PR TITLE
Making it work in oss

### PR DESCRIPTION
**Pull request by FredrikFolkesson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # File-Connectivity-Service
 
-[![CircleCI](https://circleci.com/gh/qlik-ea/core-file-connectivity-service.svg?style=shield&circle-token=55d7bdfc4f3827e260a2e3480dbd64eab52417c0)](https://circleci.com/gh/qlik-ea/core-file-connectivity-service)
+[![CircleCI](https://circleci.com/gh/qlik-core/core-file-connectivity-service.svg?style=shield&circle-token=55d7bdfc4f3827e260a2e3480dbd64eab52417c0)](https://circleci.com/gh/qlik-corea/core-file-connectivity-service)
```
Comment (wennmo) [inline README.md]: typo qlik-corea

Comment (peol) [reply README.md]: You can also remove the token

File: examples/README.md [hunk 2]
```diff
@@ -2,14 +2,14 @@
 
 This example focus on how to load data from Dropbox using the file-connectivity-service.
 
-Note that before you deploy, you must accept the [Qlik Core EULA](https://ca.qliktive.com/docs/master/beta/) by setting the `ACCEPT_EULA` environment variable.
+Note that before you deploy, you must accept the [Qlik Core EULA](https://qlikcore.com/beta/) by setting the `ACCEPT_EULA` environment variable.
 
 Start by cloning the file-connectivity-service repository with:
-- `git clone https://github.com/qlik-ea/core-file-connectivity-service.git`
+- `git clone https://github.com/qlik-core/core-file-connectivity-service.git`
```
Comment (wennmo) [inline examples/README.md]: qlik-core -> qlik-oss

Comment (peol) [inline examples/README.md]: qlik-oss

File: examples/README.md [hunk 3]
```diff
@@ -2,14 +2,14 @@
...
 - `cd file-connectivity-service`
 - `npm install`
 
-Copy the file [`airports.csv`](https://github.com/qlik-ea/core-file-connectivity-service/blob/master/data/airports.csv) located in the `/data` folder to your Dropbox.
+Copy the file [`airports.csv`](https://github.com/qlik-core/core-file-connectivity-service/blob/master/data/airports.csv) located in the `/data` folder to your Dropbox.
```
Comment (peol) [inline examples/README.md]: qlik-oss

**Pull request merged**